app/src/pages/inside/testCaseLibraryPage/testCaseList/testCaseList.tsx

```diff
@@ -66,16 +71,27 @@ export const TestCaseList = memo(
       name: {
         content: testCase.name,
         component: (
-          <TestCaseNameCell
-            priority={testCase.priority}
-            name={testCase.name}
-            tags={testCase.tags}
-          />
+          <div
+            className={cx('cell-wrapper', { selected: testCase.id === selectedTestCaseId })}
+            onClick={() => handleRowClick(testCase.id)}
```

Comment:
a11y here as well

Reply:
why we wrapping button in a button with same handler?